client/my-sites/upgrades/domain-management/email/google-apps-users-card.jsx

```diff
@@ -27,11 +28,14 @@ const GoogleAppsUsers = React.createClass( {
 		user: React.PropTypes.object.isRequired
 	},
 
-	canAddUsers() {
-		const domainsInContext = this.props.selectedDomainName
+	getDomains() {
```

Comment:
I was "inspired" by the [`getDomains` function from `domain-warnings`](https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/upgrades/components/domain-warnings/index.jsx#L65). But you do have a point - maybe something like `getDomainsAsList`?